core/src/main/java/org/ehcache/core/PersistentUserManagedEhcache.java

```diff
@@ -40,7 +40,12 @@
 import java.util.Set;
 
 /**
- * PersistentUserManagedEhcache
+ * Implementation of the {@link PersistentUserManagedCache} which is a cache with a persistent pool outside of a
```

Comment:
either "of PersistentUserManagedCache" or "of the PersistentUserManagedCache interface"